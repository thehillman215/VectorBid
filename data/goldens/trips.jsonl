{"trip_id":"EWR-73N-001-T1","pairing_id":"EWR-73N-001","day":5,"origin":"DEN","destination":"EWR"}
{"trip_id":"EWR-73N-001-T2","pairing_id":"EWR-73N-001","day":3,"origin":"LAX","destination":"EWR"}
