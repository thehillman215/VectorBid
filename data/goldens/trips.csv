trip_id,pairing_id,day,origin,destination
EWR-73N-001-T1,EWR-73N-001,5,DEN,EWR
EWR-73N-001-T2,EWR-73N-001,3,LAX,EWR
